packages/react-checkbox/src/Checkbox.Checked.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
```

Comment:
Naming nit, in most of the converged stories we're not using a period between the component name and the story name (though I actually like that convention better--easier to read)

Reply:
Let's be consistent first, this is easily changeable at one for the whole repo